AUTOTEST/machine-lassen.sh

```diff
@@ -38,7 +38,7 @@ shift
 # Basic build and run tests
 mo="-j test"
 eo=""
-roij="-ij -ams -rt -mpibind -rtol 1e-3 -atol 1e-3"
```

Comment:
@rfalgout @ulrikeyang @oseikuffuor1 At some point we should consider a new approach to do the GPU regression tests, since GPU cannot produce the same result as CPU. We now rely on these loose `tol`, in a more and more wacky way. I remember @oseikuffuor1 suggested something.  Thanks!